src/test/resources/fixtures/serialized/local-auth-by-type-10.rsf

```diff
@@ -0,0 +1,13 @@
+add-item	{"local-authority-eng":"BAS","local-authority-type":"UA","name":"Bath and North East Somerset","official-name":"Bath and North East Somerset Council","start-date":"1996-04-01"}
```

Comment:
I thought it was useful to have an example of multi item RSF in the project. When the other work is done, I was going to use the example in a functional test - hence the location. Perhaps we should allow comments in RSF!